lib/msfdb_helpers/standalone.rb

```diff
@@ -0,0 +1,67 @@
+require 'msfdb_helpers/db_interface'
+module MsfdbHelpers
+  class Standalone < DbInterface
+
+    def initialize(options:, db_conf:, connection_string:)
+      @options = options
+      @db_conf = db_conf
+      @conn = PG.connect(connection_string)
```

Comment:
Verified docker support works; Although I'm think we should clean up this stack trace when the credentials don't work:

```
 bundle exec ruby ./msfdb reinit --connection-string=postgresql://postgres:mysecretpassword@localhost:9000/postgres
Using database port 6000 found in /Users/user/.msf4/database.yml
Traceback (most recent call last):
	5: from ./msfdb:985:in `<main>'
	4: from ./msfdb:985:in `new'
	3: from /Users/user/Documents/code/metasploit-framework/lib/msfdb_helpers/standalone.rb:8:in `initialize'
	2: from /Users/user/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/pg-1.2.3/lib/pg.rb:58:in `connect'
	1: from /Users/user/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/pg-1.2.3/lib/pg.rb:58:in `new'
/Users/adfoster/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/pg-1.2.3/lib/pg.rb:58:in `initialize': could not connect to server: Connection refused (PG::ConnectionBad)
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 9000?
could not connect to server: Connection refused
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 9000?
```

Dumping the stack trace I'm somewhat okay with, but a well placed `print_error('Could not connect to standalone PostgreSQL instance. Ensure that the connection string is valid, and that the database is accessible')`, or similar, could help IMO

Reply:
Future me wished past me documented this a bit better; I set up the database with:
```
sudo docker run -p 9000:5432 -e POSTGRES_PASSWORD=mysecretpassword postgres:14.2
```

Will have to get this workflow added to the docs 📚